docs/faq.rst

```diff
@@ -125,19 +131,24 @@ Which ML libraries does Fairlearn support?
     algorithm can be evaluated using our metrics.
 
 Does Fairlearn support multi-class classification?
-    On the assessment side, Fairlearn's :class:`MetricFrame` can be used with
-    any metrics for supervised learning, including those for multi-class classification.
+    On the assessment side, Fairlearn's :class:`fairlearn.metrics.MetricFrame`
```

Comment:
you can set the `currentmodule` on top of these files to avoid having to repeat `fairlearn` here, and have it as 

```suggestion
    On the assessment side, Fairlearn's :class:`metrics.MetricFrame`
```

Reply:
Done!